### PR TITLE
Remove `component` attribute from semantic conventions

### PR DESCRIPTION
**Pull request by arminru**
In #271 it was decided to drop the span attribute `component` from the semantic conventions.

### CODE REVIEW

File: specification/data-database.md [hunk 1]
```diff
@@ -13,7 +13,6 @@ attribute names.
 
 | Attribute name | Notes and examples                                           | Required? |
 | :------------- | :----------------------------------------------------------- | --------- |
-| `component`    | Database driver name or database name (when known) `"JDBI"`, `"jdbc"`, `"odbc"`, `"postgreSQL"`. | Yes       |
```
Comment (Oberon00) [inline specification/data-database.md]: A follow-up for this is #245.

**Comment by carlosalberto** [comment]
I think we are ready to merge this one, but @arminru might need to do some merging locally first (there's no option to resolve conflicts using the github UI).

**Comment by arminru** [comment]
@carlosalberto done! (although I did press the _Update branch_ button on GH 😉)

**Comment by dyladan** [comment]
@arminru possibly you created the PR without checking the "allow maintainers to update" option checked?

**Pull request merged**